liquibase-core/src/main/java/liquibase/change/core/CreateViewChange.java

```diff
@@ -166,11 +166,11 @@ protected InputStream openSqlStream() throws IOException {
         try {
             ResourceAccessor resourceAccessor = Scope.getCurrentScope().getResourceAccessor();
 
-            String path = getPath();
             if (ObjectUtil.defaultIfNull(getRelativeToChangelogFile(), false)) {
-                path = resourceAccessor.get(getChangeSet().getChangeLog().getPhysicalFilePath()).resolveSibling(path).getPath();
+                return resourceAccessor.get(getChangeSet().getChangeLog().getPhysicalFilePath()).resolveSibling(getPath()).openInputStream();
```

Comment:
Does this (and the similar change inSQLFileChange) throw a good error when referencing a file that does not exist in this branch?

Reply:
It depends on the implementation (currently). `PathResource` throws a `FileNotFoundException` but the `S3Resource` logs a message and returns null.